File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/task93/Task93_PERPLEXITY_llama_3_sonar_large_32k_chat.java
code here:
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;
import java.util.ArrayList;
import java.util.Collections;

public class Task93_PERPLEXITY_llama_3_sonar_large_32k_chat {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new FileReader("input.txt"));
        ArrayList<String[]> records = new ArrayList<>();
        String line;
        while ((line = br.readLine()) != null) {
            records.add(line.split("="));
        }
        Collections.sort(records, (a, b) -> a[0].compareTo(b[0]));
        for (String[] record : records) {
            System.out.println(String.join("=", record));
        }
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task93_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
code here:
#include <iostream>
#include <fstream>
#include <vector>
#include <algorithm>
#include <string>

int main() {
    std::ifstream file("input.txt");
    std::vector<std::pair<std::string, std::string>> records;
    std::string line;
    while (std::getline(file, line)) {
        size_t pos = line.find("=");
        records.push_back({line.substr(0, pos), line.substr(pos + 1)});
    }
    std::sort(records.begin(), records.end());
    for (const auto& record : records) {
        std::cout << record.first << "=" << record.second << std::endl;
    }
    return 0;
}